### PR TITLE
Check that concurrency level is honored

### PR DESCRIPTION
**Pull request by lhellebr**
### Problem Statement
Concurrency has been reworked in dynflow

### Solution
Add a test for concurrency level being honored

trigger: test-robottelo
pytest: tests/foreman/cli/test_remoteexecution.py::TestAnsibleREX::test_positive_run_serial

### CODE REVIEW

File: tests/foreman/cli/test_remoteexecution.py [hunk 1]
```diff
@@ -749,6 +749,78 @@ def test_positive_run_concurrent_jobs(self, registered_hosts, target_sat):
...
+
+        :CaseLevel: System
+
+        :parametrized: yes
+        """
+        hosts = registered_hosts
+        output_msgs = []
+        template_file = f"/root/{gen_string('alpha')}.template"
+        target_sat.execute(
+            f"echo 'rm /root/test-<%= @host %>; echo $(date +%s) >> /root/test-<%= @host %>; sleep 120; echo $(date +%s) >> /root/test-<%= @host %>' > {template_file}"
```
Comment (Griffin-Sullivan) [inline tests/foreman/cli/test_remoteexecution.py:778]: what does the `<%= @host %>` part do? Is this specific to Job Templates?

**Review by Griffin-Sullivan — APPROVED** [review]
Looks good just a question about the @host syntax

File: tests/foreman/cli/test_remoteexecution.py (hunk 1 above)
Comment (lhellebr) [reply tests/foreman/cli/test_remoteexecution.py:778]: It's an ERB syntax and it outputs hostname

**Comment by pondrejk** [comment]
trigger: test-robottelo
pytest: tests/foreman/cli/test_remoteexecution.py::TestAnsibleREX::test_positive_run_serial

**Review by pondrejk — APPROVED** [review]

**Pull request merged**